umm-spec-lib/src/cmr/umm_spec/test/umm_record_sanitizer.clj

```diff
@@ -340,6 +341,15 @@
   (-> record
       (sanitize-umm-record-urls :service)))
 
+(defn sanitized-umm-t-record
+  "Place holder for the sanitizers needed for a given umm-t record."
+  [record]
+  ;; Out of 9 required fields, only ToolKeywords can get nil generated value for some reason,
```

Comment:
Why?

Reply:
That's just what the schema dictates, if I had to guess

Reply:
I really don't know. The schema says it's required and it's the same structure as the Service schema for the ScienceKeywords, where no nil values were generated...

Reply:
It has to be the schema. I just double checked and it's not ScienceKeywords in UMM-S that's supposed to be equivalent with the ToolKeywords in UMM-T. It's ServiceKeywords, and the minItems being 0 for ToolKeywords is wrong if it's required. I'm gonna ask Simon to fix it.